tests/qlayers_test.py

```diff
@@ -23,16 +23,18 @@
 from tensorflow.keras.layers import Flatten
 from tensorflow.keras.layers import Input
 from tensorflow.keras.models import Model
+from tensorflow.keras.backend import clear_session
 
-from qkeras import binary
-from qkeras import model_save_quantized_weights
 from qkeras import QActivation
+from qkeras import QDense
```

Comment:
since you are doing the reconstruction of qlayers.py, I think it is good to separate the qlayers_test.py as well accordingly in this PR. what do you think?